packages/cli/__tests__/zostso/__system__/stop/cli.zos-tso.stop.address-space.system.test.ts

```diff
@@ -104,8 +104,7 @@ describe("zos-tso start address-space", () => {
                     SYSTEM_PROPS.zosmf.host,
                     SYSTEM_PROPS.zosmf.port,
                     SYSTEM_PROPS.zosmf.user,
-                    SYSTEM_PROPS.zosmf.pass
-                ]);
+                    SYSTEM_PROPS.zosmf.password                ]);
```

Comment:
This happened because of the replace all I tired to do 😢 
Can we implement prettier? 😋